saleor/core/views.py

```diff
@@ -4,7 +4,7 @@
 
 
 def home(request):
-    products = products_with_details(request.user)[:8]
+    products = products_with_details(request.user).filter(is_featured=True)[:8]
```

Comment:
Wouldn't it make sense to add `products_for_home()`? User object should not be passed as we don't want home to display hidden products as it could lead to surprises.